packages/fxa-shared/db/models/auth/account.ts

```diff
@@ -425,4 +427,16 @@ export class Account extends BaseAuthModel {
     }
     return account;
   }
+
+  static async setMetricsOpt(
+    uid: string,
+    state: 'in' | 'out',
```

Comment:
What's the advantage of using 'in' and 'out' strings rather than a boolean all the way through?

Reply:
Yeah, I like the descriptiveness, though right the type mixing is why I asked. I think it'd be more clear (for me at least) to make `number | null` a boolean but I won't block r+ on that.

I think having the timestamp in the DB is a good call too.